tracer/test/test-applications/integrations/Samples.ServiceStack.Redis/Samples.ServiceStack.Redis.csproj

```diff
@@ -9,9 +18,15 @@
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1' OR '$(TargetFramework)'=='netcoreapp3.0'">
```

Comment:
It looks like you're setting the same version value regardless of the TargetFramework, so maybe we can remove the PackageReference changes

Reply:
The case I'm attempting to avoid with this setup is if you set the NuGet to `6.2.0` it'll cause the .NET Core 2.1/3.0 builds to fail in VS and then you can't run the tests.
So if you were wanting to test that version (or maybe higher versions?) you can just swap out the `$(ApiVersion)` in `<ItemGroup Condition="'$(TargetFramework)'!='netcoreapp2.1' AND '$(TargetFramework)'!='netcoreapp3.0'">`'s and avoid breaking the other NET Core targets. I don't think I did a good job explaining that reasoning in the comment in that csproj or maybe I'm just overthinking it

Reply:
Ohhh I see what you're saying now. Maybe a more bulletproof (maybe not as clean?) way to do this is to just override the ApiVersion if we ever detect the bad combination of 6.2.0 and netcoreapp2.1/netcoreapp3.0. To do that you could restore the csproj to its original contents and then add a second ApiVersion line like this. What do you think?

```
<ApiVersion Condition="('$(TargetFramework)'=='netcoreapp2.1' OR '$(TargetFramework)'=='netcoreapp3.0') AND '$(ApiVersion)' == '6.2.0'">6.1.0</ApiVersion>
```

Reply:
I like it, very nice 👍